tests/unit/test_exceptions.py

```diff
@@ -13,17 +13,36 @@
 # limitations under the License.
 
 from unittest import TestCase
-
+import tempfile, os
 from backup_monkey.exception import * 
+from backup_monkey import SplunkLogging
 
 class ExceptionTests(TestCase):
+    log_file = tempfile.mkstemp()[1]
+
+    @classmethod
+    def setUpClass(cls):
+        SplunkLogging.set_path(cls.log_file)
```

Comment:
Clean the log file (delete/recreate) so the tests are independent / self contained.